### PR TITLE
{bio}[GCCcore/13.3.0] modkit v0.4.1

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/415

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/modkit/modkit-0.4.1-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+description = 'A bioinformatics tool for working with modified bases.'
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/nanoporetech/modkit/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Rust', '1.78.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/modkit/modkit-0.4.1-GCCcore-13.3.0.eb:16]: @pavelToman To fix this:
```
Can't locate IPC/Cmd.pm in @INC
```
You'll need to add `Perl-bundle-CPAN` as a build dep

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
node3101.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.11.3
See https://gist.github.com/laraPPr/33faf4e2a201f0363265c2297ecd9615 for a full test report.

**Pull request merged**